e2e/components/upload/upload-surveillance/upload-surveillance.spec.js

```diff
@@ -0,0 +1,32 @@
+import UploadSurveillanceComponent from './upload-surveillance.po';
+import LoginComponent from '../../login/login.po';
+import Hooks from '../../../utilities/hooks';
+
+let hooks, loginComponent, uploadSurveillanceComponent;
+
+beforeEach(async () => {
+    uploadSurveillanceComponent = new UploadSurveillanceComponent();
+    loginComponent = new LoginComponent();
+    hooks = new Hooks();
+    await hooks.open('#/surveillance/upload');
+});
+
+describe('When uploading a surveillane activity as ONC-ACB', () => {
```

Comment:
```suggestion
describe('when uploading a surveillance activity as ONC-ACB', () => {
```

Capitalization & typo